$OpenBSD: patch-configure_in,v 1.7 2015/01/08 09:21:38 dcoppa Exp $

OpenBSD lacks 'nm -D'

setuid root/setgid utmp are handled by PLIST

Do not enable VTE support if libvte is around

--- configure.in.orig	Tue Jan  6 07:16:26 2015
+++ configure.in	Thu Jan  8 02:04:52 2015
@@ -313,7 +313,7 @@ AC_SUBST(IMAGELIB_LIBS)
 for ml_cv_lib_path in /lib /usr/lib /lib/* `echo $LIBRARY_PATH | tr ":" " "` ; do
 	libm_files=`ls $ml_cv_lib_path/libm\\.so* $ml_cv_lib_path/libm\\.sl* 2> /dev/null`
 	for libm_file in $libm_files ; do
-		if test -z "$ml_cv_libm_dir" -a -n "`$NM -D $libm_file | $GREP pow`" ; then
+		if test -z "$ml_cv_libm_dir" -a -n "`$OBJDUMP -T $libm_file | $GREP \ pow$`" ; then
 			ml_cv_libm_dir=`dirname $libm_file`
 		fi
 	done
@@ -641,39 +641,12 @@ AC_ARG_ENABLE(pty_helper,
 	pty_helper=$enable_pty_helper)
 if test "$utmp" = "yes" -a "$pty_helper" != "yes" ; then
 	UTMP_CFLAGS="-DUSE_UTMP"
-	has_utmp=`grep utmp /etc/group 2>/dev/null`
-	if test "$has_utmp" ; then
-		INSTALL_OPT="-m 2755 -g utmp"
-	else
-		INSTALL_OPT="-m 4755 -o root"
-	fi
-else
-	INSTALL_OPT="-m 755"
 fi
 
-if test "$gui" = "fb" ; then
-	INSTALL_OPT="-m 4755 -o root"
-fi
+INSTALL_OPT="-m 0555 -s"
 
 AC_SUBST(UTMP_CFLAGS)
 AC_SUBST(INSTALL_OPT)
-
-if test "${have_gtk}" = "yes" ; then
-	if test "${gtk_version}" = "3.0" ; then
-		PKG_CHECK_MODULES(VTE, vte-2.90, have_vte=yes, have_vte=no)
-		LIBVTE_SUFFIX=2_90
-		AC_SUBST(LIBVTE_SUFFIX)
-	else
-		PKG_CHECK_MODULES(VTE, vte, have_vte=yes, have_vte=no)
-	fi
-	if test "$have_vte" = "yes" ; then
-		OUTPUT_FILES="gtk/Makefile ${OUTPUT_FILES}"
-		# "tr ..." is hack for cygwin.
-		# (removing \r which is output by win32 native pkg-config.)
-		VTE_CFLAGS="`echo $VTE_CFLAGS | tr -d '\r'`"
-		AC_SUBST(VTE_CFLAGS)
-	fi
-fi
 
 tools="mlclient,mlconfig,mlcc,mlterm-menu,mlimgloader,registobmp"
 AC_ARG_WITH(tools,
